### DISCUSSION TITLE
Comfast ax?

### DISCUSSION THREAD

**Discussion by Vofi-martin** [WLAN-Pi/feedback#39] [Hardware - WLAN Pi Neo2] [answered]
Will The Neo ever support the new Comfast ax adapter?

**Comment by jiribrejcha** [comment, marked as answer]
Hi Martin,

Technically it is feasible, but it will take some significant effort. We have briefly discussed this with the team, but we haven't come to a conclusion yet.

From hardware perspective, the Wi-Fi 6/6E adapters are larger, so the Handheld case for NEO2 won't work. Unless you connect the new adapter to the bottom USB port.

How critical is this for your use case? Would it use it with a single Pi or multiple of them?

**Comment by Vofi-martin** [comment]
I have like 5 pis without case and 5 with.use the in classed so no need for
cases.

**Comment by Vofi-martin** [comment]
May take them to wlpc.
